### DISCUSSION TITLE
In mini.completion, is it possible to scroll down in the floating window?

### DISCUSSION THREAD

**Discussion by dnguyener** [nvim-mini/mini.nvim#560] [Q&A]
As an example, I've selected **run** and would like to scroll down in the floating window for additional information.

<img width="895" alt="image" src="https://github.com/echasnovski/mini.nvim/assets/12126945/9f58ee9d-cfb1-4054-9d2d-9c5f96ba9d05">

**Comment by echasnovski** [comment]
Sorry, no. This is out of scope for 'mini.completion'.
